wiki/Website-Changelog.md

```diff
@@ -0,0 +1,169 @@
+# Changelog authoring best practices
+
+- [Tips for writing consistent changelog entries](#tips-for-writing-consistent-changelog-entries)
+  - [General guidance](#general-guidance)
+  - [Consistent terminology](#consistent-terminology)
+  - [Component references](#component-references)
+  - [Content formatting](#content-formatting)
+    - [Component changes](#component-changes)
+    - [New icons](#new-icons)
+- [Templates](#templates)
+  - [Templates for npm packages](#templates-for-npm-packages)
+  - [Templates for Figma changelog entries](#templates-for-figma-changelog-entries)
+
+---
+
+## Tips for writing consistent changelog entries
+
+### General guidance
+- Changelog entries should always start with a paragraph (not a list).
+- Consider using a list instead of a long paragraph of text to communicate multiple changes for one element.
+- If you need to include a note (like a migration tip), add it to the bottom of the entry in a new paragraph. Consider using emphasized text if you need to draw more attention to it.
+
+
+- Entries should use complete sentences but be short, clear, and descriptive enough to be helpful.
+- Use past tense.
+- Entries should always end with a period.
+
+### Consistent terminology
+- Use consistent terms for different types of changes: 
+  - Bugfix: “Fixed”
+  - New component, token, variant: “Added”
+  - Update: “Changed”, “Refactored”
+  - Removed or deleted: “Removed”
+  - Other edge cases: “Upgraded”, “Reduced”, “Prevented”
+ 
+### Component references
+- When referencing components, use the plain-text component name, e.g., `Accordion`.
+- All components or token names, including those in the summary, should be enclosed in backticks (`).
+
+### Content formatting
+
+While best practice guidelines are provided, clarity is most important, so use your best judgment. 
+
+#### Component changes
+
+For component changes, place the component name first and enclose it in backticks (`), followed by a dash (-), and then a brief summary.
+
+| Do | Dont |
+|:---|:-----|
+| `Dropdown` - changed the `@height` property to use `max-height` instead of a fixed height.  | Changed the `@height` property in `Dropdown` to use `max-height` instead of a fixed height. |
+
+#### New icons
+
+For icons, use a comma-separated list instead of a bulleted list and start with the icon values. Icon names should be enclosed in backticks (`) and use the Ember value.
+
+| Do | Dont |
+|:---|:-----|
+| `twitter-x`, `twitter-x-color`, `aws-cdk`, `aws-cdk-color`, `jfrog`, and `jfrog-color` icons added. | Added a new set of service icons: <ul><li>`twitter-x`</li><li>`twitter-x-color`</li><li>`aws-cdk`</li><li>`aws-cdk-color`</li><li>`jfrog`</li><li>`jfrog-color`</li></ul> | 
```

Comment:
I think keeping the icons in a sentence format works well; for example, if we have more than three icons, the list can get long and take up more space than needed.